parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -24,6 +24,14 @@ assignFilePermissions() {
     lastlog
     waagent.log
     syslog
+    unattended-upgrades/unattended-upgrades.log
```

Comment:
These files may be added based on cluster config, so we need to prep their file perms properly